universal7885-common/universal7885-common.mk

```diff
@@ -226,8 +248,13 @@ PRODUCT_PACKAGES += \
     NfcNci \
     Tag
 
+ifeq ($(BOARD_SEPOLICY_TEE_FLAVOR),teegris)
+PRODUCT_COPY_FILES += \
+    $(LOCAL_PATH)/configs/nfc/s3nrn82.conf:$(TARGET_COPY_OUT_VENDOR)/etc/libnfc-sec-vendor.conf
+else
 PRODUCT_COPY_FILES += \
-    $(LOCAL_PATH)/configs/nfc/libnfc-sec-vendor.conf:$(TARGET_COPY_OUT_VENDOR)/etc/libnfc-sec-vendor.conf
+    $(LOCAL_PATH)/configs/nfc/s3nrn81.conf:$(TARGET_COPY_OUT_VENDOR)/etc/libnfc-sec-vendor.conf
```

Comment:
Picking nfc config based on mobicore/teegris will conflict with when/if we add m10s, a7y18lte, etc..
Then, would need to be hardcoded check based on codenames, they all have different nfcs.